Mussel 0.1.4

renatogamboa opened the pull request:
![TagCheck](https://bitrise-steplib-git-check.herokuapp.com/tag?pr=3117)

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)


**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

lpusok commented:
Hello @renatogamboa,
Noticed that there are no Step inputs and outputs provided yet.
I recommend testing  your Step (using a local or git path) as described here: https://devcenter.bitrise.io/contributors/testing-and-versioning-your-steps/#running-the-step-from-a-direct-source. Usually this test is contained in a `bitrise.yml` which can be included in your Step repository as well.
For an example see the sample workflow in the *Xcode test* Step: https://github.com/bitrise-steplib/steps-xcode-test/blob/c5c2694b75dd14dd69b1f054cdb6a8e37867766e/bitrise.yml#L37

renatogamboa commented:
Hi @lpusok , thank you for your feedback and review. I appreciate you closing the Pull Request for me as well.

I've addressed the issues from this pull request and added an `input` + `step.sh` + test as well. 
Thank you for your patience and assistance here.

New PR: https://github.com/bitrise-io/bitrise-steplib/pull/3122
